src/main/java/Duke/Tool/Emoji.java

```diff
@@ -0,0 +1,26 @@
+package Duke.Tool;
+
+/**
+ * Represents emoji symbol class.
+ */
+public enum Emoji {
```

Comment:
I really like how you separated the emojis out into a enum!

Reply:
Thank you !:)

Reply:
> I really like how you separated the emojis out into an enum!

Thank you!:)